src/organisms/TableOfContents/TableOfContents.stories.tsx

```diff
@@ -152,6 +152,28 @@ export const BorderVariant: StoryFn<StoryProps> = (args) => {
   );
 };
 
+export const BorderVariantHorizontal: StoryFn<StoryProps> = (args) => {
+  return (
+    <MemoryRouter>
+      <TableOfContentsProvider items={args.items}>
+        <Container>
+          <TableOfContents variant="borderHorizontal" />
+          <section>
+            {args.items.map((item, index) => (
+              <Section
+                key={item.value}
+                label={item.label}
+                value={item.value}
+                color={COLORS[index]}
+              />
+            ))}
+          </section>
+        </Container>
```

Comment:
I think this story should show the use case where the menu is above the sections, like it would be used in acquire.